packages/malloc/src/api.ts

```diff
@@ -9,11 +9,16 @@ export interface MemBlock {
 export interface MemPoolOpts {
     buf: ArrayBuffer;
     size: number;
+    /**
+     * Start is the anchor index inside the arraybuffer, so we can't save it inside the arraybuffer itself.
+     * If you pass the ArrayBuffer to other consumers they must use the same start value
+     */
     start: number;
     end: number;
     compact: boolean;
     split: boolean;
     minSplit: number;
+    skipInitialization: boolean
```

Comment:
Not entirely sure what that option is for? :)

Reply:
In my use case i'm creating another allocator for the same arraybuffer (on webworker for example)
So i need to have an option to prevent the constructor from setting everything to zero

Reply:
That makes sense!